modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -306,6 +308,19 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     ]);
     // #endregion
 
+    // #region reducing keychain data
+
+    userKeychain.encryptedPrv = this.bitgo.encrypt({
+      input: btoa(String.fromCharCode.apply(null, Array.from(new Uint8Array(userReducedPrivateMaterial)))),
```

Comment:
cant use Buffer.toString('base64') since it does not work as expected on the browser

Reply:
Please add this as a comment so future readers know not to refactor this.

Reply:
done